gradle.properties

```diff
@@ -1,14 +1,14 @@
 org.gradle.caching=true
 group=io.ballerina.lib
 version=0.3.0-SNAPSHOT
```

Comment:
Why not `1.0.0` ?

Reply:
Yeah, I think we can use that

Reply:
We use only a specific set of APIs from `io.confluent.kafka.schemaregistry.client.SchemaRegistryClient`, so it is not fully supported